components/brave_rewards/resources/extension/brave_rewards/background/api/locale_api.ts

```diff
@@ -22,6 +22,7 @@ export const getMessage = (message: string, substitutions?: string[]): string =>
 
 export const getUIMessages = (): Record<string, string> => {
   const strings = [
+    'backupWalletTitle',
```

Comment:
you need to add `backupWalletNotification` here as well

Reply:
Fixed, and removed translation from `en_GB`.